docs/conf.py

```diff
@@ -0,0 +1,57 @@
+"""
+Sphinx documentation builder
+"""
+
+# General options:
+import os
+import sys
+from pathlib import Path
+
+sys.path.append(os.path.abspath('../client'))
+
+project = "PurpleCaffeine"
+copyright = "2023"  # pylint: disable=redefined-builtin
+author = ""
+
+_rootdir = Path(__file__).parent.parent
+
+# The full version, including alpha/beta/rc tags
+release = "0.0.0"
+# The short X.Y version
+version = "0.0"
```

Comment:
latest :p ?

Reply:
we will need to automate that and read data from `VERSION.txt` file later

Reply:
Even from tag, nop ?